mods/win7-uac-dim.wh.cpp

```diff
@@ -0,0 +1,121 @@
+// ==WindhawkMod==
+// @id              win7-style-uac-dim
+// @name            Windows 7 Style UAC Dim
+// @description     Restores the desktop screenshot on the User Account Control screen
+// @version         1.0.0
+// @author          aubymori
+// @github          https://github.com/aubymori
+// @include         consent.exe
+// @compilerOptions -lgdi32 -lgdiplus
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# Windows 7 Style UAC Dim
+This mod restores the full desktop screenshot on the background of the secure desktop
+for UAC. Since Windows 8, it has only displayed the wallpaper.
+
+**Before**:
+
+![Before](https://raw.githubusercontent.com/aubymori/images/main/win7-uac-dim-before.png)
+
+**After**:
+
+![After](https://raw.githubusercontent.com/aubymori/images/main/win7-uac-dim-after.png)
+*/
+// ==/WindhawkModReadme==
+
+#include <windhawk_utils.h>
+#include <gdiplus.h>
+
+using namespace Gdiplus;
+
+int x = GetSystemMetrics(SM_XVIRTUALSCREEN);
+int y = GetSystemMetrics(SM_YVIRTUALSCREEN);
+int cx = GetSystemMetrics(SM_CXVIRTUALSCREEN);
+int cy = GetSystemMetrics(SM_CYVIRTUALSCREEN);
+
+HBITMAP g_hbDesktop = NULL;
+
+void (*OnPaint_orig)(HWND) = nullptr;
+void OnPaint_hook(HWND hWnd)
```

Comment:
Calling conventions are missing in the typedefs, which is likely to cause a crash for 32-bit targets. Please add calling conventions or, if the mod doesn't support 32-bit, limit it to 64-bit, which will also make compilation faster. See also:
https://github.com/ramensoftware/windhawk/wiki/Development-tips#hooks-and-calling-conventions